src/Corax/Constants.cs

```diff
@@ -38,6 +39,20 @@ public enum Operator
                 And
             }
         }
+
+        public static class Suggestions
+        {
+            public const int DefaultNGramSize = 4;
+
+            public const float JaroWinklerThreshold = 0.7f;
+
+            public enum Algorithm
```

Comment:
Why have this and `StringDistanceAlgorithm` ?

Reply:
Because the defaults of the algorithms may not be the default that RavenDB (and Lucene uses). We can use the default at the `StringDistanceAlgorithm` that we use and kill this one.